src/test/scala/com/atomist/rug/kind/csharp/CSharpFileTypeTest.scala

```diff
@@ -81,12 +81,12 @@ object CSharpFileTypeTest {
   val HelloWorldSources =
     SimpleFileBasedArtifactSource(StringFileArtifact("src/hello.cs", HelloWorld))
 
-  val HelloWorldProject = new ProjectMutableView(EmptyArtifactSource(), HelloWorldSources)
+  def helloWorldProject = new ProjectMutableView(EmptyArtifactSource(), HelloWorldSources)
```

Comment:
These changes are necessary because using a val is incorrect as `ProjectMutableView` is mutable. Later tests could see changed state.